XRTK-Core/Packages/com.xrtk.core/Tests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -705,11 +461,9 @@ public void Test_07_01_EnableServicesByType()
 
             // Add test 1 services
             MixedRealityToolkit.TryRegisterService<ITestDataProvider1>(new TestDataProvider1(testService1));
-            MixedRealityToolkit.TryRegisterService<ITestExtensionService1>(new TestExtensionService1());
```

Comment:
shouldn't this just be:

```
MixedRealityToolkit.TryRegisterService<ITestService1>(new TestService1());
```

Seems like odd that we're just removing instead of updating the usages. I'd assume it should work just the same since there wasn't really any functional difference between an extension service and a regular one.

Reply:
Renamed, although I could <blame> the original test creator for that ;D